pkg/sdk/poc/generator/show_by_id_filtering.go

```diff
@@ -0,0 +1,46 @@
+package generator
+
+type ShowByIDFiltering struct {
+	Kind            string
+	Args            string
+	IdentifierBased bool
```

Comment:
I wonder if we could use an enum instead? What if we have more showByIDs which have a different behavior?  Such flag could be confusing.

Reply:
You're right, that would probably be a better approach than just a flag